components/builder-api-client/src/lib.rs

```diff
@@ -84,6 +84,7 @@ pub struct Project {
     pub ident: String,
     pub state: String,
     pub job_id: String,
+    pub target: String,
```

Comment:
Would there be any benefit to using a PackageTarget type here instead of a String?

Reply:
In this case I don't believe so. I _think_ we could make a change like that, but not as a part of this PR. That would be a significant refactor of the way we're handling all of these pieces of data in the cli/api client.